app/panel/components/Summary.jsx

```diff
@@ -686,6 +696,29 @@ class Summary extends React.Component {
 		);
 	}
 
+	_renderRewardsIcon() {
+		const { unread_offer_ids } = this.props;
+
+		const unreadOffersAvailable = (unread_offer_ids && unread_offer_ids.length > 0) || false;
+
+		const rewardsIconClassNames = ClassNames(
+			'Summary__rewardsIcon',
+			'Summary__rewardsIcon--absolutely-positioned',
+			'g-tooltip',
+			{
+				hide: this.props.is_expert,
+			}
+		);
+
+		return (
+			<div className={rewardsIconClassNames} onClick={this.showRewardsView}>
+				<NavButton path="/detail/rewards/list" imagePath="../../app/images/panel/rewards-icon.svg" />
+				{unreadOffersAvailable && <NavButton classNames="Summary__rewardsIcon__star" path="/detail/rewards/list" imagePath="../../app/images/panel/purple-star.svg" />}
```

Comment:
I haven't been able to test this because I don't know how to generate a reward...

Reply:
As per Vinnie's decision about this, the button should display regardless of whether any rewards (new or seen) are available, so long as the user has not turned Rewards off, so it should be enough to check that the icon takes the user to the Rewards list view, and that it does not display when Rewards are off. For testing the purple star appearance / logic, see next comment.